### PR TITLE
Fix grommet theme usage

### PR DESCRIPTION
**Pull request by vujevits**
- remove fill because content does not fit on the screen
- fix footer position
- fix using all theme

### CODE REVIEW

File: packages/app/src/App.tsx [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
         <TestnetBanner />
         <Router>
           <Switch>
             <Route exact path="/" component={Home} />
             <Route path="/stakers/:id" component={Token} />
           </Switch>
         </Router>
-        <Foot />
       </Grid>
+      <Foot />
```
Comment (kyriediculous) [inline packages/app/src/App.tsx:24]: This moves the footer out of the grid which assigns xxsmall to the row for it

**Comment by kyriediculous** [comment]
fixed theme usage in https://github.com/Tenderize/tender-app/pull/37 already

File: packages/app/src/App.tsx (hunk 1 above)
Comment (vujevits) [reply packages/app/src/App.tsx]: I fixed the grid layout in a good way now on this branch, but the branch you have merged changed a whole lot of things :)

**Comment by vujevits** [comment]
@kyriediculous look at this commit: https://github.com/Tenderize/tender-app/pull/38/commits/20d53ac41e3feb29f7ca8e34e8836845e65810c1

what we are (should be) trying to achieve:
1. content scrolls
2. footer sticks to the bottom of the page, except if it's pushed lower by the content

On `master` the content is squeezed together. Also there were quite a few changes which I'm not sure if are accidental or intentional - like the changes to tabbed component, width, etc.

**Pull request closed without merging**